rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1928,38 +1915,46 @@ def test_remove_ksm_blockchain_account(rotkehlchen_api_server):
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     async_query = random.choice([False, True])
-    response = requests.get(api_url_for(
+
+    requests.get(api_url_for(
         rotkehlchen_api_server,
         'blockchainbalancesresource',
-    ))  # to populate totals
-    assert_proper_response(response)
-    ksm_totals_beginning = rotki.chain_manager.totals.assets[A_KSM].amount
-    assert ksm_totals_beginning != ZERO
+    ))  # to populate balances
     response = requests.delete(
         api_url_for(
             rotkehlchen_api_server,
             'blockchainsaccountsresource',
             blockchain=SupportedBlockchain.KUSAMA.value,
         ),
         json={
-            'accounts': [SUBSTRATE_ACC1_KSM_ADDR],
+            'accounts': [SUBSTRATE_ACC2_KSM_ADDR],
```

Comment:
Why in this test the acc changed?

Reply:
I actually just reverted this test to how it looked like before the balances query removal pr since balances update is now returned again.

Iirc I had to modify the account in that pr due to one of them had no balance but we had to check that totals change after the account removal.

Reply:
I see. Hmm I can't check the logic and test addy easily. Will just trust you on this.